components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -842,7 +842,12 @@ document.observe('xwiki:dom:loaded', function()
                  newItem.insert({top: new Element('span', {'class': "fa fa-exclamation-triangle", title: "$escapetool.javascript($services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardPhenotype'))"}).update(' ')})
                }
                newItem.__source = inputElt;
+
                var tools = new Element('span', {'class' : 'tools hint'});
```

Comment:
Let's drop the `hint` css class from here. It's from an early version of the tools, which had plain text not buttons, and it was supposed to make the font smaller and the color lighter. But now we're already overwriting the font size, and we want the default text color for the (i).